### ISSUE TITLE
[Sourcing][P1][Regression][Accessibility][Platform Table] Unable to traverse or no visible focus on platform table rows/ cells

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#11645]
#### Is this a bug, enhancement, or feature request?
Regression issue

#### Describe your proposal.
After 0.43.31 release , we are unable to traverse through table cells using screenreader.
Replicable on https://deploy-preview-11608--fundamental-ngx.netlify.app/#/platform/table/basic

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
core/ platform: 0.43.31

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Reach the table after toolbar buttons 
and notice that using up down , left right arrow keys we are unable to traverse through the rows or cells and also blue color focus border is not showing up

#### Please provide stackblitz example(s).
replicable on https://deploy-preview-11608--fundamental-ngx.netlify.app/#/platform/table/basic

https://github.com/SAP/fundamental-ngx/assets/88182358/fb035ed7-4d06-4ccd-95af-c0452a4722d5

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
This is found during ATL testing.

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by sagar7259** [comment]
Hello,

after bringing the focus on a first or random table cell, next user cannot user arrow keys to traverse within the table cells and when the tab key is used, focus comes out of the table. So basically its impossible to traverse within all tables using arrow keys.

Regards,
Sagar

**Comment by mikerodonnell89** [comment]
Fixed in 0.43.32 https://www.npmjs.com/package/@fundamental-ngx/core/v/0.43.32?activeTab=versions
